src/mscorlib/src/System/Globalization/CultureInfo.cs

```diff
@@ -156,18 +156,18 @@ internal static void AsyncLocalSetCurrentUICulture(AsyncLocalValueChangedArgs<Cu
         private static readonly bool init = Init();
         private static bool Init()
         {
-            if (s_InvariantCultureInfo == null)
-            {
-                CultureInfo temp = new CultureInfo("", false);
-                temp._isReadOnly = true;
-                s_InvariantCultureInfo = temp;
-            }
-
             s_userDefaultCulture = GetUserDefaultCulture();
             s_userDefaultUICulture = GetUserDefaultUILanguage();
             return true;
         }
 
+        private static CultureInfo GetInvariantCulture()
```

Comment:
I believe this should be a private constructor instead of a static helper method. Nothing wrong with it, but code styling wise.

Reply:
Fixed.